modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -279,6 +279,70 @@ autoSaveAsDraftTest(
 	}
 );
 
+baseTest(
+	'LPD-31427: Select web content display template with the Preview feature',
```

Comment:
Let's try to follow the [new pattern for descriptions](https://liferay.slack.com/archives/C06A59K5D46/p1721293618198129) that Echo I've tried and change it to: 
```
baseTest(
	'Select web content display template with the Preview feature',
	{
		tag: '@LPD-31427',
	},
	async ({journalEditArticlePage, page, site}) => {
```

Reply:
It works for me. Did you run it on a new unused bundle? Based on my experience the journal article editor persists some menu states between the test. For example if I opened the display template tab on the admin user and run the test the menu would be already open without opening it.